example/57-managedseedset.yaml

```
@@ -26,8 +26,9 @@ spec:
                 name: my-managed-seed-set
             spec:
               dns:
-                # "replica-name" in DNS domains will be replaced by the actual replica name
-                ingressDomain: replica-name.garden.shoot.dev.k8s-hana.ondemand.com
+                ingress:
+                  # "replica-name" in DNS domains will be replaced by the actual replica name
+                  domain: replica-name.ingress.local.seed.local.gardener.cloud
```

Cosmetic suggestion: I think the ingress domain usually is a subdomain of the Shoot's dns domain. We could avoid using local setup related domain and instead use a subdomain from the Shoot dns domain as ingress domain for the Seed.

```suggestion
                  domain: replica-name.ingress.crazy-botany.core.my-custom-domain.com
```

---

Done via: [`ef3f739` (#13572)](https://github.com/gardener/gardener/pull/13572/commits/ef3f739914975c5a3d652b5b0a2b2bbcca610897)